AnimatedComponents/src/components/floating-button/circular-floating-button/CircularFloatingButton.tsx

```diff
@@ -32,42 +32,44 @@ const CircularFloatingButton = (props: FloatingButtonProps) => {
       : withTiming(0, { duration: animationTransitionDuration }),
   )
   const { width } = Dimensions.get('window')
-  const FAB_SIZE = 54
-  const circleScale = Number((width / FAB_SIZE).toFixed(1))
-  const circleSize = circleScale * FAB_SIZE
-  const dist = circleSize / 2 - FAB_SIZE * 1.5
-  const radius = dist * 1.5
+  const iconSize = 54
```

Comment:
can we not make it dynamic as per user requirement? given this comment earlier as well

Reply:
No, this is not possible. We have to make it static.